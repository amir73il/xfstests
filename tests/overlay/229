#! /bin/bash
# SPDX-License-Identifier: GPL-2.0
# Copyright (C) 2017 CTERA Networks. All Rights Reserved.
#
# FS QA Test 229
#
# Run unionmount testsuite to do a quick sanity test of a snapshot mount
# with multiple snapshots without verifying correctness of the snapshot
#
seq=`basename $0`
seqres=$RESULT_DIR/$seq
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1	# failure is the default!
trap "_unionmount_testsuite_cleanup; exit \$status" 0 1 2 3 15

# get standard environment, filters and checks
. ./common/rc
. ./common/filter

# remove previous $seqres.full before test
rm -f $seqres.full

# real QA test starts here

_supported_fs overlay
_supported_os Linux
_require_overlay_snapshot
_require_unionmount_testsuite sn

_unionmount_testsuite_run --sn=10 --samefs

# success, all done
echo "Silence is golden"
status=0
exit
