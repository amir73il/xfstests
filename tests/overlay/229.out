QA output created by 229
Silence is golden
